src/main/java/org/folio/circulation/domain/representations/FeeFineActionStorageRepresentation.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.domain.representations;
+
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.Account;
+import org.folio.circulation.domain.FeeFine;
+import org.folio.circulation.domain.FeeFineOwner;
+import org.folio.circulation.domain.User;
+import org.folio.circulation.support.JsonPropertyWriter;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineActionStorageRepresentation extends JsonObject {
+  public FeeFineActionStorageRepresentation(Account account, FeeFineOwner feeFineOwner,
+    FeeFine feeFine, Double amount, Double balance, User loggedInUser) {
+    super();
+
+    if (account == null || feeFine == null || feeFineOwner == null) {
+      return;
+    }
+
+    this.put("id", UUID.randomUUID().toString());
+    this.put("userId", account.getUserId());
+    this.put("accountId", account.getId());
+    this.put("source", loggedInUser == null ? "" : String.format("%s, %s",
+      loggedInUser.getLastName(), loggedInUser.getFirstName()));
```

Comment:
For which purpose this field is going to be used? Can we use an UUID here instead?

Reply:
This field is shown to the user on FE
![Screen Shot 2020-03-06 at 19 03 39 1](https://user-images.githubusercontent.com/14072374/76105374-857a0900-5fdd-11ea-9312-fb7e1799428d.png)

Reply:
Typically, the `source` field in the UI is populated using the change `metadata` for a record. Is this the case for fee / fine actions?

If so, this is likely populated automatically by the RAML Module Builder when the record is created, based upon the token or user header passed from `Okapi` to `mod-feesfines` and `mod-circulation` does not need to be involved.

Reply:
Oh, it appears that this is done in a non-standard way in [mod-feefines](https://github.com/folio-org/mod-feesfines/blob/6abd57690aa51df7eaffd6ca3d33073818c0bed1/ramls/feefineactiondata.json#L40) :-/

Reply:
Yeah, apparently. Looks like it has a lot of non-standard features.